### ISSUE TITLE
BlockLauncher doesn't appear to work on 0.15.7

### ISSUE THREAD

**Issue by Black-DeathPE** [zhuowei/MCPELauncher#1409]
Hi, this might just be a thing with my device or something but BlockLauncher no longer works on MCPE 0.15.7. It can load into safe mode, but even if all modding features (including patches and texture packs) are off and I turn off safe mode it crashes on launch.

Device: SHIELD Tablet
Android version: Marshmallow (6.0.1)
BlockLauncher version: 1.13.1

**Comment by zedman2000** [comment]
There is a beta out.

**Comment by SaccyScriptor** [comment]
Yes indeed BL 1.13 was the most broken one for the entire history of BL that I remember. Anyway, it is updated now and will not yield this same crash anymore. I had the very same problem before. Please close this issue as it is fixed now.
